src/routes/layout.model.ts

```diff
@@ -0,0 +1,13 @@
+export type Routes = { routes: (RouteWithHref | RouteWithSections)[] };
```

Comment:
Das verwendest du nicht und daher könnte die ganze Datei gelöscht werden. Oder du stellst die Routen genauso wie in #14 per `load()` für die Routen bereit.

Reply:
War copy & paste aus #14 aber hab nicht reingeschaut